ar3:
  stepper_config:
    # Unconfigured joints have a 1 because 0 would result in divide by 0 error
    deg_per_step:
      - 0.0225
      - 0.0128
      - 0.0375 
      - 0.045 
      - 0.18
      - .01 # Stepper not working currently

    step_delay: 170